services/storage/docker/boot.sh

```diff
@@ -26,11 +26,14 @@ then
   echo "  PIP :"
   $SC_PIP list | sed 's/^/    /'
 
+  #------------
+  echo "  setting entrypoint to use watchmedo autorestart..."
+  entrypoint='watchmedo auto-restart --recursive --pattern="*.py" --'
```

Comment:
Does this allow to remote ssh with vscode, work on the files and have immediate restart?

Reply:
yes. @sanderegg  found the tool and I am adding it everywhere in development mode